pttrack/templates/pttrack/patient_detail.html

```diff
@@ -172,49 +172,49 @@ <h4 class="panel-title"><a data-toggle="collapse" href="#collapse2">Uploaded Pre
   			<div class="panel panel-default">
   				{# followup_set is a custom method in the patient object returns a list rather than a queryset, so no .all() #}
     			<div class="panel-heading">
-      				<h4 class="panel-title"><a data-toggle="collapse" href="#collapse3">Followups ({{ total_followups}})</a></h4>
+      				<h4 class="panel-title"><a data-toggle="collapse" href="#collapse3">{% trans 'Followups' %} ({{ total_followups}})</a></h4>
     			</div>
     			<div id="collapse3" class="panel-collapse collapse">
       				{% for note in patient.followup_set %}	
 						<div class="panel-body">
 							{# here, the url takes an arugment to route it to the correct  #}
-							<p><a href="{% url 'followup' pk=note.pk model=note.type %}"><strong>{{ note.type }} Followup:</strong></a> {{ note.short_text }}</p>
-							<p class="text-muted text-right">by {{ note.author }} ({{ note.author_type }}) at {{ note.written_datetime }}</p>
+							<p><a href="{% url 'followup' pk=note.pk model=note.type %}"><strong>{{ note.type }} {% trans 'Followup' %}:</strong></a> {{ note.short_text }}</p>
+							<p class="text-muted text-right">{% trans 'by' %} {{ note.author }} ({{ note.author_type }}) {% trans 'at' %} {{ note.written_datetime }}</p>
```

Comment:
When we have a {% trans 'by' %} do we ever have to worry about the word "by" translating to different things? Like a note by me is "von mir" but if you say you're standing by me you stand "neben mir". See what I mean?

Reply:
Yes, that's right! As I've mentioned in another comment there is a "context"-option, that helps to solve this problem. This should be used whenever there are words with multiple translations.